pkg/check/node_perf.go

```diff
@@ -0,0 +1,207 @@
+package check
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/vmware/govmomi/vim25/methods"
+	"github.com/vmware/govmomi/vim25/mo"
+	"github.com/vmware/govmomi/vim25/types"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+)
+
+// CheckNodeDiskPerf Checks node performance of master nodes.
+type CheckNodeDiskPerf struct{}
+
+func (c *CheckNodeDiskPerf) Name() string {
+	return "CheckNodePerf"
```

Comment:
I wonder if this check should be differently named for master and worker nodes. I know we aren't alerting on worker nodes right now, but say we do that in future then to alerting rules both of the checks will appear the same and we won't have ability to change severity or messaging of alerts from master nodes vs worker nodes. For example:

if we want an alert that says - "Detected more than 100ms io latency on master node" , we can't do that if check is named same for both master and worker.